app/scripts/helpers/Settings.js

```diff
@@ -1,10 +1,12 @@
 (function () {
 
-	function getCurrentUrl(){
-		if (typeof document != "object") return "127.0.0.1";
-	  var domain = document.URL.match(/:\/\/(.[^/:]+)/)[1];
-      return domain;
-	}
+    function getCurrentUrl(){
+        if (typeof document !== 'object' || typeof process === 'object')
```

Comment:
Hey @pboyer , I'm not sure why do we need typeof document !== 'object', I believe it's for determining of NWK mode. But I always get this equal to true in NWK. I found that typeof process === 'object' works fine for NWK.
Maybe you can explain why do we need first condition in here